webpack/dev.ts

```diff
@@ -40,8 +44,17 @@ export const getDevConfig = (target: Target): Configuration => {
         })
       ],
       new DefinePlugin({
-        CONFIG_LOG_LEVEL: JSON.stringify('DEBUG')
-      })
+        CONFIG_LOG_LEVEL: JSON.stringify('DEBUG'),
+        CONFIG_PERMISSION_HOSTS: JSON.stringify({
+          origins: ['http://*/*', 'https://*/*']
+        }),
+        CONFIG_ALLOWED_PROTOCOLS: JSON.stringify(['http:', 'https:'])
+      }),
+      profiler &&
+        new BundleAnalyzerPlugin({
+          analyzerMode: 'server',
+          analyzerPort: target === 'firefox' ? 8888 : 8889
+        })
```

Comment:
We were earlier using profiler in production only. Should add in both profiles I think. Then we can add it to `webpack/plugins.ts` and pass the flag there.